internal/services/config/config_test.go

```diff
@@ -0,0 +1,266 @@
+// Copyright 2019 Sorint.lab
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"testing"
+	"os"
+	"io/ioutil"
+	"path"
+	errors "golang.org/x/xerrors"
```

Comment:
move this to the end of the import in a separate block. The logic is: first block go stdlib packages, second block: agola packages, third block: external packages.